sql/hive/v0.12.0/src/main/scala/org/apache/spark/sql/hive/Shim12.scala

```diff
@@ -242,6 +242,14 @@ private[hive] object HiveShim {
     }
   }
 
+  // make getConvertedOI compatible between 0.12.0 and 0.13.1
+  def getConvertedOI(inputOI: ObjectInspector,
+                     outputOI: ObjectInspector,
+                     equalsCheck: java.lang.Boolean =
```

Comment:
let's remove the parameter, to keep it as the same method signature within the `HiveShim` 0.13.1